### PR TITLE
Support text decorations

### PR DESCRIPTION
**Pull request by dnfield**
Fixes https://github.com/dnfield/vector_graphics/issues/145

### CODE REVIEW

File: packages/vector_graphics/lib/src/listener.dart [hunk 1]
```diff
@@ -450,16 +450,33 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
...
+    final List<ui.TextDecoration> decorations = <ui.TextDecoration>[];
+    if (decoration & 0x1 != 0) {
+      decorations.add(ui.TextDecoration.underline);
+    }
+    if (decoration & 0x2 != 0) {
+      decorations.add(ui.TextDecoration.overline);
+    }
+    if (decoration & 0x4 != 0) {
+      decorations.add(ui.TextDecoration.lineThrough);
+    }
```
Comment (jonahwilliams) [inline packages/vector_graphics/lib/src/listener.dart:468]: Consider making these named constants

Comment (dnfield) [reply packages/vector_graphics/lib/src/listener.dart:468]: I almost wish they were exposed from the `dart:ui` class but will do.

Comment (dnfield) [reply packages/vector_graphics/lib/src/listener.dart]: Done

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**